### PR TITLE
chore: release 4.4.0

### PR DESCRIPTION
**Pull request by fireridlle**
- Improve types (errors.pargeJSON, errors.TooManyRequests, `signal` for client.query options)
- Fix false positive cases for ClientClosed error
- Option to disable checking driver version (new Client({checkNewVersion: false}))
- Fix PageHelper honors size with map/filter
- Fix module issue for `Vitessedge`

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ var notifyAboutNewVersion = util.notifyAboutNewVersion()
  *   Only applicable for NodeJS environment (when http2 module is used). Default is 500ms;
  *   can also be configured via the FAUNADB_HTTP2_SESSION_IDLE_TIME environment variable
  *   which has the highest priority and overrides the option passed into the Client constructor.
- * @param {?boolean} options.notifyAboutNewVersion
+ * @param {?boolean} options.checkNewVersion
  *   Enabled by default. Will print a message to terminal if installed version is ahead the latest one
```
Comment (faunaee) [inline src/Client.js:169]:
```suggestion
 *   Enabled by default. Prints a message to the terminal when a newer driver is available.
```

**Review by faunaee — APPROVED** [review]
One minor code comment suggestion.

Otherwise, LGTM! 👍

**Review by faunaee — APPROVED** [review]
LGTM! 👍

**Review by rodrimaia — APPROVED** [review]
image

**Pull request merged**